molecule/templates/rakefile.j2

```diff
@@ -8,44 +8,22 @@ require 'fileutils'
 task spec: 'serverspec:all'
 task default: :spec
 
-env = YAML.load_file('{{ molecule_file }}')['vagrant']
-
-if env.nil?
-	puts "No vagrant instances defined. "
-	instances = {}
-else
-	instances = env['instances']
-end
-instances_to_test = {}
-instances.each do |instance|
-  instance_name = instance['name']
-  if instance.key?('options')
-    if instance['options'].key?('append_platform_to_hostname')
-      if instance['options']['append_platform_to_hostname']
-        instance_name = instance['name'] + '-{{ current_platform }}'
-      end
-    end
-  end
-  instances_to_test[instance_name] = instance
-end
+hosts = YAML.load_file('{{ state_file }}')['hosts']
```

Comment:
We still template the location of the state file and the directory under which to look for serverspec tests.